### PR TITLE
Improve pagination for openqa-mon

### PR DESCRIPTION
**Pull request by grisu48**
Add some minor improvements to the pagination feature:

* Display page count from the beginning onwards
* Add handlers for home, end, page up, page down and arrow left/right keys
* Use case instead of nested ifs in the openqa-revtui key handler

Follow-up of https://github.com/os-autoinst/openqa-mon/pull/157, in preparation of a Release-Candidate

### CODE REVIEW

**Comment by grisu48** [comment]
@ilmanzo please have a look, if you have time :slightly_smiling_face:

File: cmd/openqa-mon/openqa-mon.go [hunk 1]
```diff
@@ -881,51 +881,71 @@ func continuousMonitoring(remotes []Remote) {
...
-			tui.SetHideStates(!tui.DoHideStates())
-		case 'p':
-			config.Paused = !config.Paused
-			if config.Paused {
-				SetStatus()
-			} else {
+		p[2], p[1], p[0] = p[1], p[0], b
+
+		// Handle special keys
+		if p[2] == 27 && p[1] == 91 && p[0] == 72 { // home
```
Comment (ilmanzo) [inline cmd/openqa-mon/openqa-mon.go]: how about handle all of the `91 ; 27` special keycodes inside a single if and make a switch inside ?
something like 
```go
if p[1]==91 && p[2]==27 { // special key pressed
  switch p[0] {
    case 70: // end
    case 53: // pgup
    case 54: // pgdn
    case 68: // left arrow
    case 67: // right arrow
  }
}
```

**Comment by ilmanzo** [comment]
Overall very good, left only a small comment for better readability :)

**Comment by ilmanzo** [comment]
Nit: would be great to avoid a screen refresh/update when the page content does not actually change. 

For example when you are on the first page and press Left or page up, then the screen flickers to update itself but it's really not needed. Maybe we can just use a `dirty` flag to signal the `Update()` function it's time to redraw or not ? 

```go
func (tui *TUI) PrevPage() {
	tui.Model.mutex.Lock()
	defer tui.Model.mutex.Unlock()
    if tui.currentPage > 0 {
	  tui.currentPage--
      // here we need to refresh
      tui.dirty=true
    }
}
```  
but we can just bring this to another issue, possibly related also to #159

File: cmd/openqa-mon/openqa-mon.go (hunk 1 above)
Comment (grisu48) [reply cmd/openqa-mon/openqa-mon.go]: Good point! I've updated it accordingly.

**Comment by grisu48** [comment]
> Nit: would be great to avoid a screen refresh/update when the page content does not actually change.
> 
> For example when you are on the first page and press Left or page up, then the screen flickers to update itself but it's really not needed. Maybe we can just use a `dirty` flag to signal the `Update()` function it's time to redraw or not ?
> 
> ```go
> func (tui *TUI) PrevPage() {
> 	tui.Model.mutex.Lock()
> 	defer tui.Model.mutex.Unlock()
>     if tui.currentPage > 0 {
> 	  tui.currentPage--
>       // here we need to refresh
>       tui.dirty=true
>     }
> }
> ```
> 
> but we can just bring this to another issue, possibly related also to #159

We can, but so far screen refreshes are cheap on a local terminal, so I didn't bothered much.

**Review by ilmanzo — APPROVED** [review]

**Pull request merged**